modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,298 @@
+// Module included in the following assemblies:
+//
+// * machine_management/
+// * machine_management/
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-node-custom-partition_{context}"]
+= Optional: Adding a new {op-system} worker node with a custom `/var` partition in AWS
+
+{product-title} supports partitioning devices during installation by using machine configs that are processed during the bootstrap. However, if you use `/var` partitioning, the device name must be determined at installation and cannot be changed. This means that you cannot add different instance types as nodes if they have different device naming schema. For example, if you configured the `/var` partition with the default AWS device name for `m4.large` instances, `dev/xvdb`, you cannot directly add an AWS `m5.large` instance, as it uses the `/dev/nvme1n1` device by using only a new machine set.   
+
+The procedure in this section shows how to add a new {op-system-first} compute node with an instance that uses a different device name from what was configured at installation. You will create a custom user data secret and configure a new machine set. These steps are specific to an AWS cluster.
+
+.Procedure 
+
+. Change to the `openshift-machine-api` namespace:
++
+[source,terminal]
+----
+$ oc project openshift-machine-api
+----
+
+. Create a new secret from the the `worker-user-data` secret: 
+
+.. Export the `userData` section of the secret to a text file: 
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get secret worker-user-data --template='{{index .data.userData | base64decode}}' | jq > userData.txt
+----
+
+.. Edit the text file to add the `storage`, `filesystems`, and systemd` stanzas for the partitions you want to use for the new node. You can specify any link:https://coreos.github.io/ignition/configuration-v3_3/[Ignition configuration parameters] as needed.
++
+[NOTE]
+====
+Do not change the `ignition.config.version` stanza. 
```

Comment:
They probably shouldn't change the `config/merge` and `security/tls/CAs` below either